README.md

```diff
@@ -78,3 +79,52 @@ applications are running the functional tests can be run with
 `make run`
 
 (or you can substitute `local` with the name of whatever `config/*.sh` environment file you're using.
+
+
+## Setting up your local environment with Docker Compose to run functional tests against
+
+Setting up your local environment and database to run the functional tests against can be a pain. Wouldn't it be nice if you could get all the apps up and running, backed by a database in the correct state with just one command? That's hopefully what we can do with Docker Compose. It uses the latest tagged Docker images of the apps as well as a DB dump of your choosing to achieve this.
+
+### Setup
+
+  - Have Docker installed on your machine - see [here](https://www.docker.com/docker-mac)
+  - Have Docker Compose installed on your machine. Docker for Mac comes with it so you probably won't need to do anything extra. You can check with `docker-compose -v`. If you don't have it, go [here](https://docs.docker.com/compose/install/)
+  - Have a database dump that you want to use. The most up to date you can get your hands on the better. This is the data you're going to be running tests against so it should have all frameworks up to date.
+  - In the root of the functional tests repo create a directory called sql and place your database dump in there. Make sure the dump has an `.sql` suffix.
+  - Execute `$ make docker-up`
+  - Enter your AWS MFA code when prompted.
+  - If this is the first time you've run this command, the Postgres image will need to import the data from the dump. This will take a minute or two. When it's done, the apps will load, beginning with the api. When the logs in your terminal have calmed down and tell you that frontend apps have spawned uWSGI workers, move on.
+  - Go to [`http://localhost`](http://localhost). You should see the Digital Marketplace homepage.
+  - You now need to index your services. You can do this with a script in the scripts repo. From the root of your scripts repo, you're probably going to want to do this:
+        ./scripts/index-services.py  dev --api-token=myToken --search-api-token=myToken  --frameworks=g-cloud-9
```

Comment:
It'd have to work on everyones system though. If there scripts repo is on a different relative path it'd break, right?